wallet-new/bench/Client/Cardano/Wallet/Web/Endpoint/NewPayment.hs

```diff
@@ -0,0 +1,61 @@
+-- | Function for running a client, for @NewPayment@.
+
+module Client.Cardano.Wallet.Web.Endpoint.NewPayment
+    ( newPaymentIO
+    ) where
+
+import           Universum
+
+import           Client.Cardano.Wallet.Web.Api     (newPayment)
+import           Client.Cardano.Wallet.Web.Run     (runEndpointClient)
+import           Client.Cardano.Wallet.Web.Analyze (analyzeResponseIfNeeded, checkResponse)
+import           Bench.Cardano.Wallet.Types        (BenchEndpoint (..), CompleteConfig (..),
+                                                    Wallet (..), WalletAccount (..),
+                                                    WalletsConfig (..), Response,
+                                                    ResponseReport (..))
+import           Bench.Cardano.Wallet.Random       (pickRandomElementFrom,
+                                                    -- pickTwoRandomElementsFrom,
+                                                    pickRandomValueBetween)
+import           Pos.Wallet.Web.ClientTypes        (Addr, CId (..), CTx (..))
+import           Pos.Client.Txp.Util               (InputSelectionPolicy (..))
+import           Pos.Core                          (mkCoin)
+
+-- | Run 'NewPayment' client. As a result
+-- we will get a newly created transaction.
+newPaymentIO :: CompleteConfig -> IO ()
+newPaymentIO conf@CompleteConfig {..} = do
+    let passPhrase = Nothing
+        policy     = OptimizeForSecurity
+        minAmount  = 300000 :: Word64 -- In Lovelaces
+        maxAmount  = 900000 :: Word64 -- In Lovelaces
```

Comment:
My understanding is that `newPayment` benchmark will include both:
 * successfull payments
 * unsuccessfull payments because we are spending money that we don't have in our account

I believe this might give us a false results for some edge cases. If account has sufficient money - everything will be fine though (I mean - inpact of unsuccessfull payments will be small). 

So my question is: if benchmark is trying to send money from an account that doesn't have money - will this result be included or excluded from `newPayment` end results? (ps, maybe it doesn't matter at all - if average time for successfull payment is similar to average time of unsuccessful sending - but I am not sure if that's true.)

Reply:
> So my question is: if benchmark is trying to send money from an account that doesn't have money - will this result be included or excluded from newPayment end results? 

Yes, it will be included in the result, because it's just IO-action, `gauge` knows nothing about successful or unsuccessful payments. ;-)

Of course, by default it's assumed that we _have_ money we want to send, because we want to measure correct full payment.

> (ps, maybe it doesn't matter at all - if average time for successfull payment is similar to average time of unsuccessful sending - but I am not sure if that's true.)

No, average time for successful payment takes more time than unsuccessful one.